src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.tsx

```diff
@@ -197,7 +201,7 @@ export const ArtworkSidebarCurrentBidInfoFragmentContainer =
       fragment ArtworkSidebarCurrentBidInfo_artwork on Artwork {
         sale {
           is_closed: isClosed
-          is_live_open: isLiveOpen
+          isLiveOpenHappened
```

Comment:
not sure why we are doing this dance with renaming to undercase here? Figured we can use isLiveOpenHappened directly? Should I clean up other ones or there is a reason we need this conversion here?